apps/api/src/app/environments-v1/usecases/output-renderers/render-email-output.usecase.ts

```diff
@@ -50,7 +56,19 @@ export class RenderEmailOutputUsecase {
   }
 
   @Instrument()
-  private transformForAndShowLogic(body: string, fullPayloadForRender: FullPayloadForRender) {
+  private transformMailyDynamicBlocks(body: string, fullPayloadForRender: FullPayloadForRender) {
     return this.expandEmailEditorSchemaUseCase.execute({ emailEditorJson: body, fullPayloadForRender });
   }
 }
+
+export const stringifyDataStructureWithSingleQuotes = (value: unknown, spaces: number = 0): string => {
```

Comment:
@djabarovgeorge can you please elaborate with an example? Why do we need this extra handling on single quotes?

Reply:
A few reasons:  
1. Alignment with the Liquid.js client used in the framework.  
2. I noticed that when a variable is missing, the framework currently parses it as `unknown` (we have it today in the framework). This option causes this behavior. I believe it's an issue, but I couldn't fix it quickly, so I suggest we address it separately.  
For now, we've ensured consistent behavior across channel steps.